### PR TITLE
Add snippet arrayLcm

### PR DESCRIPTION
**Pull request by kriadmin**
## Description

Add a snippet arrayLcm to calculate lcm of an array. 

## What does your PR belong to?
- [ ] Website
- [x] Snippets
- [ ] General / Things regarding the repository (like CI Integration)

## Types of changes
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] Enhancement (non-breaking improvement of a snippet)
- [x]  New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the code style of this project.
- [x]  My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have checked that the changes are working properly
- [x] I have checked that there isn't any PR doing the same
- [x] I have read the **CONTRIBUTING** document.

### CODE REVIEW

File: snippets/arrayLcm.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+Use `Array.reduce()` and the `lcm` formula (uses recursion) to calculate the lowest  common multiple  of an array of numbers.
+
+```js
+const arrayLcm = arr =>{
+  const gcd = (x, y) => !y ? x : gcd(y, x % y);
+  const lcm = (x, y) => (x*y)/gcd(x, y) 
+  return arr.reduce((a,b) => lcm(a,b));
+}
+// arrayLcm([1,2,3,4,5]) -> 60
+// arrayLcm([4,8,12]) -> 12
```
Comment (iamsoorena) [inline snippets/arrayLcm.md]: I think the answer to second one is `24`

Comment (kingdavidmartins) [reply snippets/arrayLcm.md]: Agreed it is 24

**Comment by Chalarangelo** [comment]
@kingdavidmartins based on the fact that we have `arrayGcd`, `arrayMin` and `arrayMax`, I would say this is fine. They serve different use-cases.

**Comment by kingdavidmartins** [comment]
@Chalarangelo I was actualy about to open an issue, Yea I noticed. I then thought why have arrayMax() which only takes an array & breaks when arrayMax(2, 5, 454, 23, 231, 2343)? Are we going to make a new function/snippet to handle the other use case? or can't we just add a line & have this current snippet and the like handle both arr's, x amounts of values & etc

**Comment by Chalarangelo** [comment]
@kingdavidmartins we should open an issue and deal with that as a general situation, the retroactively make the changes to existing snippets! 😉

**Comment by kingdavidmartins** [comment]
@Chalarangelo Considered it done!! ⚡️ 😄

**Comment by Chalarangelo** [comment]
I'm merging this, as the only issue is what I discussed with @kingdavidmartins and that should be dealt with separately!

**Pull request merged**